src/classiceditorui.js

```diff
@@ -9,9 +9,15 @@
 
 import ComponentFactory from '@ckeditor/ckeditor5-ui/src/componentfactory';
 import FocusTracker from '@ckeditor/ckeditor5-utils/src/focustracker';
+import {
+	expandToolbarConfig,
```

Comment:
I didn't get this function name at first. Maybe something more literal like `getItemsFromConfig`? It's in `toolbar/utils` anyway, so we can drop the "toolbar" word. The same about the second function name.